### ISSUE TITLE
Unable to download mods

### ISSUE THREAD

**Issue by Jamiska89** [Hermsi1337/docker-ark-server#97]
Hello, downloaded and fired up the newest version of compose .xml you have. Server boots up and everything is nice but modding is not possible.  Changed only some minor details like server name, password, etc. The mods wont work even if dropped directly to the modding folder (ShooterGame/Content/mods). 

> Mod "2357644511" was not successfully downloaded
> Mod "2357644511" download failed

**Comment by DevPrestigious** [comment]
Did you ever get this figured out?

**Comment by Jamiska89** [comment]
Sorry, checked the docker file and seems like we did not manage to get mods to work.

**Comment by DevPrestigious** [comment]
Hey, I figured it out. Mods not downloaded properly, its not a server issue. Actually mods get registered per game account. You should go to ark workshop here and subscribe to mods
https://steamcommunity.com/app/346110/workshop/
Mods get installed on the players device and not on the server. Although you need the docker file to enable the mods to work on the server. 

Furthermore, it seems that you don't actually need to subscribe to mods, it should happen automatically when you try to join a server. However if it doesn't then you can add the mods via workshop.
Finally if you're on steam deck, make sure you have background downloads added so you can download while you're in the ark server menus.
Tl;dr, ignore the error on the server file logs, and just sign in and play. If it doesn't work consider subscribing to mods on the workshop.

**Comment by DanKGooGLy** [comment]
The issue definitely wasn't related to whether or not you were signed in to the docker image, as I tested signing in to my own steam account on multiple occasions while being subscribed to the mods.  

According to arkmanager/ark-server-tools#1281, the issue was fixed in version 1.6.65, so [I created a pull request](https://github.com/Hermsi1337/docker-ark-server/pull/107) to solve the issue.
